src/wasm.rs

```diff
@@ -223,11 +224,11 @@ where
         block: &BlockInfo,
         sender: Addr,
         msg: WasmMsg,
-    ) -> AnyResult<AppResponse> {
+    ) -> StdResult<AppResponse> {
         self.execute_wasm(api, storage, router, block, sender.clone(), msg.clone())
-            .context(format!(
-                "Error executing WasmMsg:\n  sender: {sender}\n  {msg:?}"
-            ))
+        // .context(format!(
+        //     "Error executing WasmMsg:\n  sender: {sender}\n  {msg:?}"
+        // ))
```

Comment:
Maybe it makes sense here to do something like this to not lose the information?
```suggestion
        .map_err(|e| StdError::msg(format!("Error executing WasmMsg:\n  sender: {sender}\n  {msg:?}\n{e}")))
```

Reply:
Yes, indeed, changed to into_iter().

Reply:
Rigth! Mepped this error and fixed tests!